### PR TITLE
avoid bool coercion in `DeferredActionQueues#schedule`

### PR DESCRIPTION
**Pull request by bekzod**

### CODE REVIEW

File: lib/backburner/queue.ts [hunk 1]
```diff
@@ -139,6 +129,16 @@ export default class Queue {
     return false;
   }
 
+  public push(target, method, args, stack) {
```
Comment (rwjblue) [inline lib/backburner/queue.ts]: Why is this moved around? Seems unrelated to the stated changes...

File: lib/index.ts [hunk 2]
```diff
@@ -122,7 +122,6 @@ export default class Backburner {
     @return instantiated class DeferredActionQueues
   */
   public begin(): DeferredActionQueues {
-    let options = this.options;
```
Comment (rwjblue) [inline lib/index.ts]: Seems unrelated?

Comment (bekzod) [reply lib/index.ts]: will make separate PR for it :P

File: lib/backburner/queue.ts (hunk 1 above)
Comment (bekzod) [reply lib/backburner/queue.ts]: just moved it `push`  and `pushUnique` next to each other :P, does it worth another PR

Comment (rwjblue) [reply lib/backburner/queue.ts]: ya, at least a separate commit so its easier to track why things were juggled around

Comment (bekzod) [reply lib/backburner/queue.ts]: done

**Comment by rwjblue** [comment]
Awesome, thanks!
